pages/tabs.py

```diff
@@ -0,0 +1,13 @@
+from utils.code_and_show import example_app
```

Comment:
similar comments need to be address, based on the ones form the checklist-plots.py file inside pages